test/regression/sql/standard_conforming_strings.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE foo(t text);
+
+set standard_conforming_strings = off;
+select * from foo where t = 'foo\'bar';
```

Comment:
Should we insert this value into the table and the following 'SELECT' statement can get the correct result from the table. Rest LGTM.